sources/force-render-png.sh

```diff
@@ -91,6 +93,10 @@ for COMPONENT in $COMPONENTS; do
 				# Renders every sif file present 
 				NAME=${sample%.*}
 				EXT=${sample##*.}
+				if [ -f "$CURRENT_DIR/$NAME.txt" ]; then
+					EXP_VERSION=`cat "$CURRENT_DIR/$NAME.txt"`
+					get-synfig $EXP_VERSION
+				fi
```

Comment:
I suggest to remove line 98 and move lines 85-87 after this line 99.

Reply:
I just realized that we don't need line 85-87, since https://github.com/synfig/synfig-tests-regressions/blob/77ebdf4a530faa6d083761af7a0912f9e5804dac/sources/force-render-png.sh#L26 and https://github.com/synfig/synfig-tests-regressions/blob/77ebdf4a530faa6d083761af7a0912f9e5804dac/sources/force-render-png.sh#L35 already handle this while line 85-87 was just handling for tar balls.

I'll test and will update you.